tabbycat/users/views.py

```diff
@@ -79,6 +79,16 @@ def get_form_kwargs(self):
     def get_success_url(self):
         return reverse_tournament('options-tournament-index', self.tournament)
 
+    def form_valid(self, form):
+        form.save(
+            domain_override=self.request.get_host(),
+            use_https=self.request.is_secure(),
+            request=self.request,
+        )
+        messages.success(self.request, "Successfully invited user to create an account for the tournament.")
```

Comment:
Mark for translation:
```suggestion
        messages.success(self.request, _("Successfully invited user to create an account for the tournament."))
```